# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/cssom-view/scroll-behavior-element.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/cssom-view/scroll-behavior-element.html.ini
--- a/testing/web-platform/meta/css/cssom-view/scroll-behavior-element.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/cssom-view/scroll-behavior-element.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -10,10 +10,10 @@
 
   [Element with smooth scroll-behavior ; scrollTo() with auto behavior]
     expected:
-      if (os == "win") and (processor == "x86_64") and not fission and not swgl and not debug and not webrender: ["PASS", "FAIL"]
-      if (os == "win") and (processor == "x86_64") and not fission and not swgl and debug and webrender: ["PASS", "FAIL"]
-      if (os == "linux") and webrender: ["PASS", "FAIL"]
-      if (os == "mac") and debug: ["PASS", "FAIL"]
+      if not webrender and (os == "win") and not debug and (processor == "x86_64"): ["PASS", "FAIL"]
+      if webrender and (os == "win") and debug and not swgl: ["PASS", "FAIL"]
+      if webrender and (os == "mac") and debug: ["PASS", "FAIL"]
+      if webrender and (os == "linux"): ["PASS", "FAIL"]
 
   [Element with smooth scroll-behavior ; scroll() with default behavior]
     expected:
@@ -29,7 +29,9 @@
 
   [Element with smooth scroll-behavior ; scrollIntoView() with auto behavior]
     expected:
-      if (os == "linux") and debug and webrender and not fission: ["PASS", "FAIL"]
+      if not swgl and (os == "linux") and debug and webrender and not fission: ["PASS", "FAIL"]
+      if swgl and (os == "linux"): ["PASS", "FAIL"]
+      if swgl and (os == "win"): ["PASS", "FAIL"]
 
   [Element with smooth scroll-behavior ; scroll() with auto behavior]
     expected: